boards/arm/bl654_usb/Kconfig.defconfig

```diff
@@ -26,24 +26,14 @@ config FLASH_LOAD_OFFSET
 config USB_CDC_ACM
 	default n if USB_DEVICE_BLUETOOTH
 
-config BL654_USB_SERIAL_BACKEND_CDCACM
```

Comment:
It is not just shortcut, please move this option to boards/arm/bl654_usb/Kconfig, or drop this patch and assign an issue to me, thanks.

Reply:
thanks, moved to Kconfig.